README.md

```diff
@@ -1,3 +1,3 @@
-# Roly Poly
+# PrismLauncher bot
 
-PolyMC Bot for [Discord](https://discord.gg/ZeN2sncGEn).
+PrisimLauncher Bot for [Discord](https://discord.gg/prismlauncher).
```

Comment:
```suggestion
PrismLauncher Bot for [Discord](https://discord.gg/prismlauncher).
```

Reply:
It's better to keep a non-vanity invite, in case we eventually lose the level 3 or anything

Reply:
Hopefully the server will gain partner, and it's unlikely that it'll loose vanity any time soon with 120 boosts...